src/app/features/collectibles/ordinals.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Grid, Stack, Text, color } from '@stacks/ui';
+
+// import { useQuery } from '@tanstack/react-query';
+import { openInNewTab } from '@app/common/utils/open-in-new-tab';
+import { Caption } from '@app/components/typography';
+
+import { useGetOrdinalsQuery } from './hooks/useGetOrdinalsQuery';
+
+interface Props {
```

Comment:
```suggestion
interface OrdinalsProps {
```

Reply:
I saw this was fixed in last push, but now reverted back to nameless prop?

Reply:
I pushed the wrong commit, back to `OrdinalProps` now :)